src/careamics/utils/metrics.py

```diff
@@ -4,7 +4,7 @@
 This module contains various metrics and a metrics tracking class.
 """
 
-from typing import Union
+from typing import Optional, Union
```

Comment:
At some point , https://github.com/juglab/MicroSSIM?tab=readme-ov-file#usage needs to be added here. In evaluation I use MicroSSIM, MicroS3IM and range invariant PSNR. Not for now though.

Reply:
Good point. Just a small question @ashesh-0: do you just need it at evaluation time or also for validation?

Reply:
It is only needed at evaluation time.